cpp/include/Ice/Object.h

```diff
@@ -63,7 +63,7 @@ namespace Ice
         /// @private
         void _iceD_ice_ping(IncomingRequest&, std::function<void(OutgoingResponse)>);
 
-        /// Gets the Slice interfaces supported by this object as a list of type IDs.
+        /// Gets the Slice interfaces supported by this object as a list of Slice type IDs.
```

Comment:
Elsewhere in this file, we specifically say `Slice type IDs`, not just `type IDs`.
Even in the `@return` here we say Slice.